bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -856,8 +856,6 @@ spec:
                 - name: RELATED_IMAGE_VELERO_PLUGIN_FOR_GCP
                   value: quay.io/konveyor/velero-plugin-for-gcp:oadp-1.4
                 - name: RELATED_IMAGE_VELERO_PLUGIN_FOR_CSI
-                  value: quay.io/konveyor/velero-plugin-for-csi:oadp-1.3
-                - name: RELATED_IMAGE_KUBEVIRT_VELERO_PLUGIN
```

Comment:
Looks like the wrong name was removed. 858 above should be removed, this one should stay.

Reply:
Agreed; wrong line removed. :sunglasses: 

FWIW, `make bundle` appears to do the correct thing when I run it locally

```
-                - name: RELATED_IMAGE_VELERO_PLUGIN_FOR_CSI
+                - name: RELATED_IMAGE_KUBEVIRT_VELERO_PLUGIN
```

Reply:
Resolved in https://github.com/openshift/oadp-operator/pull/1407/commits/bfc3f5e75954999bad1067af7f165177fd01cdbf